cmd/commento/main.go

```diff
@@ -6,6 +6,7 @@ import (
 	"time"
 
 	_ "github.com/mattn/go-sqlite3"
+	"github.com/joho/godotenv"
 	
 	. "github.com/adtac/commento/lib"
```

Comment:
I know this isn't part of this commit, but dot imports are not nice. From [golang.org](https://golang.org/doc/go1compat): __We do not recommend the use of import . outside of tests, and using it may cause a program to fail to compile in future releases.__